Irina/src/main/java/oop/AdminUser.java

```diff
@@ -0,0 +1,27 @@
+package oop;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class AdminUser extends AnyUser {
+    private String adminUsername;
+
+    @Override
+    public String getName() {
+        return adminUsername;
+    }
+
+    @Override
+    public void setName(String name) {
+        this.adminUsername = name;
+    }
```

Comment:
If the logic remains unchainged, there is no need to Override a method, you already inherit this logic from parent class

Reply:
Method setName - removed; getName - kept